exercises/practice/simple-linked-list/.meta/config.json

```
@@ -25,4 +26,4 @@
   },
   "source": "Inspired by 'Data Structures and Algorithms with Object-Oriented Design Patterns in Ruby', singly linked-lists.",
   "source_url": "https://web.archive.org/web/20160731005714/http://brpreiss.com/books/opus8/html/page96.html"
-}
+}
```

```suggestion
}

```

JSON files must end in a new line.

---

Ah, that seems to have been the VSCode auto-formatter